packages/genesys-spark-components/src/components/stable/gux-tabs/gux-tab/gux-tab.tsx

```diff
@@ -86,6 +86,7 @@ export class GuxTab {
           'gux-tab': true,
           'gux-active': this.active
         }}
+        disabled={this.guxDisabled}
         type="button"
         aria-disabled={this.guxDisabled.toString()}
```

Comment:
Because of this attribute I think not disabling the button was a choice we made before. Maybe this is not a bug.

Reply:
I think it is intentional to be able to use the arrow keys to focus disabled tabs. For example, truncated text in tabs is shown on focus with the keyboard. Without being able to focus disabled tabs, truncated text can not be viewed by keyboard only users

Reply:
Makes sense 👍  The only other thing I am wondering is should the the `gux-tooltip` not have disabled stylings ? In `v3` it does not have disabled stylings. I can create an issue for this if so.